.github/workflows/forge-pfn.yaml

```
@@ -126,7 +126,7 @@ jobs:
   # Measures PFN latencies with a constant TPS (with network chaos)
   run-forge-pfn-const-tps-network-chaos:
     if: ${{ github.event_name != 'pull_request' }}
```

```suggestion
    if: ${{ github.event_name != 'pull_request' && always() }}
```

I think we want the next job to start regardless of success criteria. (same for all)

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-not-requiring-successful-dependent-jobs

---

@ibalajiarun, I did wonder about that, but is that not logically equivalent to the current thing? e.g., `if X && True == X`?

---

Aah, found this: https://github.com/actions/runner/issues/491#issuecomment-660122693. You're correct 😄 What a weird world.